src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseHeaderReader.cs

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Justin Van Patten. All rights reserved.
```

Comment:
See the discussion here: https://github.com/dotnet/coreclr/pull/1329#discussion_r36218185

This has been done elsewhere in coreclr and corefx repos.